Systematics/test/workspaceStd.py

```diff
@@ -750,7 +750,8 @@
 
 #pythia8 has an unanticipated EM showering feature, check have two photons from hard scatter
 process.penultimateFilter= cms.Sequence()
-if customize.processId == "th_125": # for this sample the filter removes also H -> ZG
+if customize.processId.count("ggF_X") or customize.processId.count("ttH"):
```

Comment:
going to change ggF_X --> WWgg since we also want to remove Dalitz events from non resonant samples